lint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 yarn solhint 'contracts/**/*.sol'
 if [ ! -z "$(find test/ -name "*.js")" ];
```

Comment:
actually a lot of test files are not check linting. It should be
```bash
yarn prettier -c test/*
```

Reply:
This setting will make it fail in travis, and also since the *.js tests aren't linted. So I changed it to apply to the *.ts files only.